addons/l10n_be/data/account.account.template.csv

```diff
@@ -1,187 +1,426 @@
-"a7501","Revenus des obligations","7501","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a7502","Revenus des créances à plus d'un an","7502","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a751","Produits des actifs circulants","751","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a752","Plus-values sur réalisations d'actifs circulants","752","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a753","Subsides en capital et en intérêts","753","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a754","Différences de change","754","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a755","Ecarts de conversion des devises","755","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a756","Produits des autres créances","756","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a757","Escomptes obtenus","757","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_financing","False"
-"a7600","Produits exceptionnels","7600","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_investing","False"
-"a7710","Régularisations d'impôts dus ou versés","7710","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_operating","False"
-"a7711","Régularisations d'impôts estimés","7711","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_operating","False"
-"a780","Prélèvements sur les impôts différés","780","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_operating","False"
-"a789","Prélèvements sur les réserves immunisées","789","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","account.account_tag_operating","False"
-"a790","Bénéfice reporté de l'exercice précédent","790","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","","False"
-"a791","Prélèvement sur le capital et les primes d'émission","791","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","","False"
-"a793","Perte à reporter","793","account.data_account_type_revenue","l10n_be.l10nbe_chart_template","","False"
+"a000","Company creditors, beneficiaries of third party guarantees","000","account.data_account_off_sheet","l10n_be.l10nbe_chart_template","","False"
+"a001","Third party guarantees on behalf of the company","001","account.data_account_off_sheet","l10n_be.l10nbe_chart_template","","False"
```

Comment:
root accounts (olderly known as type view) shouldn't be imported, as people aren't supposed to use them (they must use lower level accounts)

Reply:
Yeah, I was asked to add all these off sheet accounts by AVW and TSB